docs/src/app/components/pages/components/DropDownMenu/README.md

```diff
@@ -0,0 +1,6 @@
+## Drop Down Menu
+
+To find out more about the `DropDownMenu` component please visit the Material Design's
+specifications **WHERE IS IT IN THE SPEC?!**.
```

Comment:
@oliviertassinari Yes I've wondered the same thing since I started using this library. In fact, the only think select field does is wrap DropDownMenu. We'll have to address this after #2555 in my opinion